isy-bedienkonzept-doc/modules/ROOT/pages/bedienkonzept.adoc

```diff
@@ -2063,3 +2174,60 @@ Die folgende Liste zeigt einige negative Bezeichnungen und ihre neutralen Äquiv
+=== Read-Only vs. Disabled
+
+Read-Only Zustände und Disabled habe klar definierte Unterschiede die gezielt genutzt werden können.
+
+*_"Read Only" State_*
+Ein "Read Only"-Feld erlaubt es, die darin enthaltenen Daten zu sehen, jedoch nicht zu ändern. Dies ist besonders dann nützlich, wenn Informationen relevant sind, jedoch nicht modifiziert werden sollen.
+
+*Merkmale:*
+
+* Sichtbarkeit: Der Nutzende kann den Inhalt des Feldes vollständig sehen und kopieren.
+* Interaktivität: Die Daten können nicht bearbeitet werden, jedoch bleiben Funktionen wie das Markieren und Kopieren erhalten.
+* Design: Optisch wird das <<image-eingabefeldreadonly, Eingabefeld>> leicht modifiziert mit leichteren Farben oder Formen dargestellt.
+
+*Anwendungsfälle:*
+
+* Bestätigungsseiten: Nach der Eingabe eines Formulars kann ein "Read Only"-Zustand verwendet werden, um die eingegebenen Daten zur Überprüfung anzuzeigen.
+* Informationsanzeige: Felder, die wichtige Daten enthalten, die jedoch durch den aktuellen Benutzer nicht verändert werden sollen (z.B. automatisch generierte ID-Nummern, erfasste Zeitstempel).
+
+*Handlungsempfehlungen:*
+
+Verwende "Read Only" für Felder, deren Inhalte für den Benutzer relevant sind, aber nicht verändert werden dürfen.
+Gestalte das Design so, dass die visuelle Unterscheidung klar, aber nicht störend ist – z.B. durch subtile Farbveränderungen.
+Erlaube den Benutzern weiterhin, Inhalte aus dem "Read Only"-Feld zu kopieren, falls sie die Daten anderweitig benötigen.
+
+*_"Disabled" State_*
+Ein "Disabled"-Feld hingegen ist vollständig inaktiv. Der Benutzende kann den Inhalt nicht oder schlecht sehen, nicht bearbeiten und auch keine Interaktion (wie Kopieren) durchführen.
+Diese Option wird verwendet, um dem Benutzer zu signalisieren, dass bestimmte Funktionen oder Eingaben derzeit nicht zur Verfügung stehen.
+
+*Merkmale:*
+
+* Sichtbarkeit: Oft wird der Inhalt des Feldes ausgegraut oder gar nicht sichtbar gemacht.
+* Interaktivität: Keine Interaktion möglich – weder das Bearbeiten, noch das Kopieren von Inhalten.
+* Design: Meist visuell hervorgehoben durch ausgegraute Felder oder abweichende Farbschemata, um den Zustand klar zu kommunizieren.
+
+*Anwendungsfälle:*
+
+* Zugriffssteuerung: Felder, die nur von Benutzenden mit bestimmten Rechten bearbeitet werden dürfen, können für andere Nutzende "disabled" sein.
+* Bedingte Formularfelder: Wenn bestimmte Eingaben erst aktiviert werden sollen, nachdem vorherige Felder korrekt ausgefüllt wurden, können diese initial "disabled" sein.
+
+*Handlungsempfehlungen:*
+
+Nutze "Disabled" für Felder, die dem Benutzer derzeit nicht zugänglich sein sollen, um Fehlinteraktionen zu verhindern.
+Vermeide "Disabled" für wichtige Informationen, die der Benutzer sehen muss – in solchen Fällen ist "Read Only" besser geeignet.
+Mache den "Disabled"-Status klar ersichtlich, indem du eine deutliche visuelle Abgrenzung durch Ausgrauen oder andere farbliche Mittel verwendest.
+
+*Vergleich*
```

Comment:
ggf. hier mit Unterkapiteln arbeiten?
```suggestion
[[vergleich-readonly-disabled]]
==== Vergleich zwischen Readonly und Disabled
```

Reply:
Habe ich umbenannt in Vergleich und Fazit und in die *_Kapitel_* Ebene eingerückt